API/SearchUser.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
What's going in on here? The file name is SearchUser... but we are searching for... well "userName like $contactName"?

If this file is responsible for searching a user, we should search for user inside of User table, not Contacts table. Moreover, since each userName is "unique" (we enforce uniqueness ourselves) we have to look for "SPECIFIC" username, without wildcards